Gopkg.toml

```diff
@@ -90,9 +90,13 @@
   name = "github.com/spf13/cobra"
   version = "~v0.0.1"
 
+[[override]]
+  name = "github.com/json-iterator/go"
+  revision = "f2b4162afba35581b6d4a50d3b8f34e33c144682"
+
 [[constraint]]
   name = "github.com/spf13/viper"
-  version = "~v1.0.0"
+  version = "~v1.0.2"
```

Comment:
Looks like `viper` doesn't mind making breaking changes between bugfix versions... Could you pin this to a given version (i.e. `=v1.0.2` here)

Reply:
yeah :( - I meant to pin, fixing that now 🙏